Answers.md

```diff
@@ -0,0 +1,7 @@
+1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+> The differences between descibe and it is that Describe will be the entire Test Suite for test cases to live in. The it referes to each seperate test case that you create for that specific test suite.
+1. What is the point of `Test Driven Development`? What do you think about this approach?
+> The point of the Test Driven Development is to understand how the product works before codeing it. If you understand and know the corner cases of your product then you can develop more cleanly makeing sure it aglins with the test you created. In my honest opion Test Driven Development can be bennifical but it may not work for all cases. There are specific jobs just for QA personel who do Automated test. Developers in Agile work enviorments have to deploy fast. And somtimes dont have enough time to spend writing the test vs writing the code. Hence the need for QA personel to write the test to make sure it does what it suppose to do. So while this is great for smaller pieces of code 'Unit Test' a whole integration and regression this would not be idea.
```

Comment:
Great breakdown of TDD Stefan 👍 You give a good explanation and blend it well with your thoughts on the techniques.